docs/collections.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
whyyy :)

Reply:
Well it significantly improves performance, obviously.